components/elements/cards/cardevent.vue

```diff
@@ -0,0 +1,36 @@
+<template>
```

Comment:
Same here about the file name:
So maybe: `/cards/card-job.js `
I let you judget